python-avd/pyavd/_cv/client/studio.py

```diff
@@ -322,8 +305,7 @@ async def get_studio_inputs_with_path(
         try:
             response = await client.get_one(request, metadata=self._metadata, timeout=timeout)
         except Exception as e:  # pylint: disable=broad-exception-caught
```

Comment:
Since you are only interested in this type, we can improve this:
```suggestion
        except GRPCError as e:
```
And then just check the status.not_found.

Reply:
Done

```diff
         client = InputsServiceStub(self._channel)
         try:
             response = await client.get_one(request, metadata=self._metadata, timeout=timeout)
-        except Exception as e:  # pylint: disable=broad-exception-caught
-            if isinstance(e, GRPCError) and e.status == Status.NOT_FOUND:
+        except GRPCError as e:
+            if e.status == Status.NOT_FOUND:
                 # Ignore this error, since it simply means we no inputs are in the studio so we will return the default value.
                 return default_value
             raise
+        except Exception as e:
+            msg = f"Unable to get Inputs '{input_path}' of Studio '{studio_id}' in workspace '{workspace_id}'."
+            raise CVClientException(msg) from e
 
         if response.value.inputs is not None:
             return json.loads(response.value.inputs)
```

Reply:
why do you need the broad exception here? Shouldn't that be handled by the decorator?